webpack-common.js

```diff
@@ -107,34 +107,43 @@ export function getRules({
   ];
 }
 
-export function getPlugins({ excludeOtherAppLocales = true } = {}) {
+export function getPlugins({
+  excludeOtherAppLocales = true,
```

Comment:
Just wondering, while you are in here, do we still need `excludeOtherAppLocales`?

Reply:
I don't think so. We can remove it maybe.